source/Libraries/SteamLibrary/SteamShared/MetadataProvider.cs

```diff
@@ -451,6 +468,16 @@ internal string ParseDescription(string description)
             return metadata;
         }
 
+        private static IEnumerable<string> GetCompaniesFromAppDetails(List<string> companies)

```

Comment:
This looks unnecessary to me, you can replace this with: `publishers = metadata.StoreDetails.publishers?.Where(a => !a.IsNullOrWhiteSpace())`

Also, are they really including empty publishers and developers? Is that check even necessary?

Reply:
I'm not sure to be honest. The extension currently checks that so I kept it for safety

https://github.com/JosefNemec/PlayniteExtensions/blob/044ba2ab70891833b1dfb7c25998b71f7f57e390/source/Libraries/SteamLibrary/SteamShared/MetadataProvider.cs#L304-L309

Reply:
Changed to use linq in https://github.com/JosefNemec/PlayniteExtensions/pull/175/commits/f360f8e03e16abf85c54a30ecaedcfb651711c3d